docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<p>Is the tripple-quoted string per cell a done thing that I'm not privy to? I thought usually we'd use line comments (or use the markdown cell above to describe)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>It's a personal convention I use that no one has explicitly called out yet :) The idea is to have a 'cell docstring' just as modules have docstrings - read this line and you know what the cell is doing (with inline comments where needed.) It also switches up the comment style and colors and looks more appealing imo. That said, it can be overkill for short codeblocks such as here. </p>